app/org/catalyst/slackservice/db/AnalyticsHandler.java

```diff
@@ -1,9 +1,5 @@
 package org.catalyst.slackservice.db;
 
 public interface AnalyticsHandler {
-    void incrementMessageCounts(AnalyticsKey key);
-    void incrementIgnoredMessageCounts(AnalyticsKey key);
-    void incrementLearnMoreMessageCounts(AnalyticsKey key);
-    void incrementCorrectedMessageCounts(AnalyticsKey key);
-    void setTeamName(String teamId, String teamName);
+    void setTeamName(String teamId, String teamName); // TODO: is this specific to analytics and can it be removed?
```

Comment:
yup, it's specific to analytics. If we don't need to capture the workspace name for analytics, we can remove this method and any references to it. I can also move it to the token handler interface.